### DISCUSSION TITLE
Setting up OVPN(.com) provider: "NB: Remember to not specify .ovpn as part of the config name."

### DISCUSSION THREAD

**Discussion by vikekh** [haugene/docker-transmission-openvpn#2736] [General]
I get this error "NB: Remember to not specify .ovpn as part of the config name." and then stop when trying to set up `OVPN` as provider on `latest` image.

My config is:

```
- OPENVPN_PROVIDER=OVPN
- OPENVPN_CONFIG=standard.us.new-york.udp
- OPENVPN_USERNAME=myUsernameForLoggingIntoOvpn.com
- OPENVPN_PASSWORD=myPasswordForLoggingIntoOvpn.com
```

There are also `OVPN_` settings that I'm not sure how to use together with `OPENVPN_` settings? Thank you in advance!

**Comment by ilike2burnthing** [comment]
https://haugene.github.io/docker-transmission-openvpn/provider-specific/#ovpn
